themes/cardinal_service/pdb_components/opportunities_list/src/Components/Filters.jsx

```diff
@@ -54,6 +54,20 @@ const MoreFilterWrap = styled.div`
   }
 `;
 
+const MoreFilterBtn = styled.button`
+  background: transparent;
+  color: #006cb8;
+  text-decoration: underline;
+  font-weight: 600;
+
+  &:hover,
+  &:focus {
+    background: transparent;
+        color: #2e2d29;
+        box-shadow: none;
```

Comment:
Inconsistent indentation.
```suggestion
    background: transparent;
    color: #2e2d29;
    box-shadow: none;
```